src/BinaryParsers/PEBinary/PEBinary.cs

```diff
@@ -45,6 +46,12 @@ public class PEBinary : BinaryBase
             this.symbolPath = symbolPath;
             this.Valid = this.PE.IsPEFile;
             this.tracePdbLoad = tracePdbLoad;
+
+            if (this.tracePdbLoad)

```

Comment:
>if (this.tracePdbLoad) [](http://example.com/codeflow?start=12&length=22)

Please first see the description of the PR in browser, 
I notice screenshot does not show in CodeFlow, thanks.